tests/phpunit/tests/post/wpTrashPost.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @group post
+ * @covers ::wp_trash_post
+ */
+class Tests_Post_wpTrashPost extends WP_UnitTestCase {
+	/**
+	 * @var WP_Post
+	 */
+	protected $post;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->post = $this->factory()->post->create_and_get(
+			array(
+				'post_status' => 'draft',
+			)
+		);
+	}
+
+	/**
+	 * @ticket 58392
+	 * @covers ::wp_trash_post
```

Comment:
```suggestion
	 * Tests that wp_trash_post() returns a WP_Post object
	 * and sets the correct post meta to trash a post.
	 *
	 * @ticket 58392
	 *
	 * @covers ::wp_trash_post
```
- Let's add descriptions to each of the tests, starting with "Tests that" per convention.
- I also spaced out the `@ticket` and `@covers` annotations based on [the structure/grouping of annotations in this ticket](https://core.trac.wordpress.org/ticket/56070).